### PR TITLE
[Op][FX] Add `relax.clip` Op and support MobileNetV2 Importing

### PR DESCRIPTION
**Pull request by Hzfengsy**
This PR enables importing MobileNetV2 from PyTorch FX, which including the following changes:

- Add `relax.clip` Op, with PrimValue support
- Support `torch.clamp` and `torch.ReLU6` in PyTorch FX frontend
- Support `torch.nn.functional.adaptive_avg_pool2d` in PyTorch frontend

### CODE REVIEW

**Comment by Hzfengsy** [comment]
cc @MasterJH5574 @spectrometerHBH

File: python/tvm/relax/transform/legalize_ops.py [hunk 1]
```diff
@@ -826,15 +834,17 @@ def __init__(self, mod: IRModule, customize_legalize_map: Dict[str, LegalizeFunc
 
             def _convert_op(self, call: Call) -> Expr:
                 if call.op.name in self.legalize_map:
-                    # We only transform the op calls with known shape values
-                    if not all(
-                        [has_known_shape_value(arg.struct_info) for arg in call.args]
-                    ) or not has_known_shape_value(call.struct_info):
+                    try:
+                        return self.legalize_map[call.op.name](self.builder_, call)
+                    except Exception as e:  # pylint: disable=broad-except
```
Comment (MasterJH5574) [inline python/tvm/relax/transform/legalize_ops.py]: I think it is good to have clear requirement on shape here, as this is the prerequisite for a CallNode to be legalized. We can have the try-catch here to capture other potential exceptions, but it is good to keep the shape check here.

**Review by MasterJH5574 — COMMENTED** [review]
Just one minor comment, thanks!

**Review by MasterJH5574 — APPROVED** [review]

**Comment by MasterJH5574** [comment]
Thanks @Hzfengsy. Would you mind sending
* the BlockBuilder fix and tests
* the clip op
* the TOPI fix

to tlc or mainline correspondingly? That would be super helpful.

**Pull request merged**